app/src/components/map/components/DrawControls.tsx

```diff
@@ -78,6 +78,9 @@ export interface IDrawControlsProps {
   confirmDeletion?: boolean;
 }
 
+/**
+ * @deprecated Prefer using `DrawControls2` moving forward.
```

Comment:
Can we elaborate here on why someone may prefer using one over the other?

Reply:
I've added to the comment on this component, hopefully if makes sense